bazel/envoy_build_system.bzl

```diff
@@ -0,0 +1,39 @@
+# References to Envoy external dependencies should be wrapped with this function.
+def envoy_external_dep_path(dep):
+    return "//external:%s" % dep
```

Comment:
Since introduction of named Bazel workspaces, there is barely no reason for using `bind` and `//external`, IMHO.